File: Easy/2026-04-24-2833-Furthest-Point-From-Origin/solution.js
var furthestDistanceFromOrigin = function(moves) {
    let leftMoves = 0;
    let rightMoves = 0;
    let underscoreCount = 0;

    for (let move of moves) {
        if (move === 'L') {
            leftMoves++;
        } else if (move === 'R') {
            rightMoves++;
        } else if (move === '_') {
            underscoreCount++;
        }
    }

    return Math.abs(leftMoves - rightMoves) + underscoreCount;
};

File: Easy/2026-04-24-2833-Furthest-Point-From-Origin/solution.java
class Solution {
    public int furthestDistanceFromOrigin(String moves) {
        int leftMoves = 0;
        int rightMoves = 0;
        int unknownMoves = 0;

        for (char move : moves.toCharArray()) {
            if (move == 'L') {
                leftMoves++;
            } else if (move == 'R') {
                rightMoves++;
            } else if (move == '_') {
                unknownMoves++;
            }
        }

        int maxDistance = Math.abs((leftMoves + unknownMoves) - rightMoves);
        maxDistance = Math.max(maxDistance, Math.abs(leftMoves - (rightMoves + unknownMoves)));

        return maxDistance;
    }
}